src/pages/iou/IOUDetailsModal.js

```diff
@@ -0,0 +1,159 @@
+            chatReportID: PropTypes.string,
+
+            // iouReportID passed via route /iou/:iouReportID
+            iouReportID: PropTypes.string,
+        }),
+    }).isRequired,
+
+    /* Onyx Props */
+    // Holds data related to IOU view state, rather than the underlying IOU data.
+    iou: PropTypes.shape({
+        // Is the IOU Report currently being settled
+        loading: PropTypes.bool,
+
+        // Error message, empty represents no error
+        error: PropTypes.string,
+    }),
+
+    // IOU Report data object
+    iouReport: PropTypes.shape({
+        // ID for the chatReport that this IOU is linked to
+        chatReportID: PropTypes.number,
+
+        // Manager is the person who currently owes money
+        managerEmail: PropTypes.string,
+
+        // Owner is the person who is owed money
+        ownerEmail: PropTypes.string,
+
+        // The IOU transactions
+        transactions: PropTypes.arrayOf(PropTypes.shape(iouTansactionPropTypes)),
+
+        // Is the IOU report settled?
+        hasOutstandingIOU: PropTypes.bool,
+    }).isRequired,
+
+    // Session info for the currently logged in user.
+    session: PropTypes.shape({
+        // Currently logged in user email
+        email: PropTypes.string,
+    }).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+class IOUDetailsModal extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            // Temporarily placeholder, as the Venmo/Paypal issue will introduce settlement types.
+            settlementType: 'Elsewhere',
+        };
+
+        this.performIOUSettlement = this.performIOUSettlement.bind(this);
+    }
+
+    componentDidMount() {
+        // Fetch the iouReport without linking it to the chatReport. Else, the chatReport's iouReportID field would
+        // wrongly be pointed to an old settled iouReport, overwriting the iouReportID of the open iouReport.
+        fetchIOUReportByID(this.props.route.params.iouReportID, this.props.route.params.chatReportID);
+    }
+
+    performIOUSettlement() {
+        settleIOUReport({
+            chatReportID: this.props.route.params.chatReportID,
+            reportID: this.props.route.params.iouReportID,
+            paymentMethodType: this.state.settlementType,
+        });
+    }
+
+    render() {
+        const sessionEmail = lodashGet(this.props.session, 'email', null);
+        const reportIsLoading = _.isUndefined(this.props.iouReport);
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title="Details"
+                    onCloseButtonPress={Navigation.dismissModal}
+                />
+                {reportIsLoading ? <ActivityIndicator color={themeColors.text} /> : (
+                    <View style={[styles.flex1, styles.justifyContentBetween]}>
+                        <ScrollView contentContainerStyle={{flexGrow: 1, paddingStart: 20, paddingEnd: 20}}>
```

Comment:
Are these inline styles or display options for the `ScrollView`? If they are inline styles, I think they should be moved to our styles file. I see other ScrollViews are referencing styles rather than setting them inline.

Reply:
I'm not actually sure, to be honest.  I believe they are display options as they must be passed into the ScrollView via this prop, rather than being set as styles in the regular way.

Reply:
https://reactnative.dev/docs/scrollview#contentcontainerstyle

Reply:
I've moved the container styles to a new styles object 👍